### PR TITLE
Update manage-feedback-policies-in-teams.md

### PR DESCRIPTION
**Pull request by jhwang-ms**
Added "Suggest a feature" to the parameters section per PM, Gagan Dhillon.

### CODE REVIEW

File: Teams/manage-feedback-policies-in-teams.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ To turn off and turn on the features, set the following parameters:
  - **Screenshots**: Use the **AllowScreenshotCollection** flag to add screenshot collection opt-in for users.
  - **Email**: Use the **AllowEmailCollection** flag to add an email field.
  - **Log collection**: Use the **AllowLogCollection** flag to add log collection opt-in for users. Log collection is currently enabled only on mobile. For more details on what data is shared via logs, [learn more](https://go.microsoft.com/fwlink/?linkid=2168178).
+ - **Suggest a feature**: Set the allowTeamsFeedbackPortal parameter to enabled to allow users who are assigned the policy to suggest a feature. Setting the parameter to disabled turns off the feature and users who are assigned the policy dont have the option to give feedback. The default setting is taken from your M365 Optional Connected Experiences policy setting. To learn more about that setting, see [Overview of optional connected experiences in Office](https://learn.microsoft.com/deployoffice/privacy/optional-connected-experiences).
```
Comment (JohanFreelancer9) [inline Teams/manage-feedback-policies-in-teams.md:69]:
```suggestion
 - **Suggest a feature**: Set the **AllowTeamsFeedbackPortal** parameter to enabled to allow users who are assigned the policy to suggest a feature. Setting the parameter to disabled turns off the feature and users who are assigned the policy don't have the option to give feedback. The default setting is taken from your Microsoft 365 optional connected experiences policy setting. To learn more about that setting, see [Overview of optional connected experiences in Office](https://learn.microsoft.com/deployoffice/privacy/optional-connected-experiences).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jhwang-ms, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@jhwang-ms I don't see the AllowTeamsFeedbackPortal parameter in [Set-CsTeamsFeedbackPolicy](https://learn.microsoft.com/en-us/powershell/module/skype/set-csteamsfeedbackpolicy?view=skype-ps).

**Comment by jhwang-ms** [comment]
@MikePlumleyMSFT have to check with the PM owner; I was just the messenger for this PR.

**Pull request merged**